x-pack/plugin/esql/qa/testFixtures/src/main/resources/date_nanos.csv-spec

```diff
@@ -1249,3 +1249,26 @@ sv_nanos:date_nanos               | a:keyword             | b:keyword  | c:keywo
 2023-03-23T12:15:03.360103847Z | 2023-03-23T12:15:03.360Z | 2023-03-23 | 2023-03-23T12:15:03.360103847Z
 2023-03-23T12:15:03.360103847Z | 2023-03-23T12:15:03.360Z | 2023-03-23 | 2023-03-23T12:15:03.360103847Z
 ;
+
+Date Nanos Date Diff
+required_capability: date_nanos_date_diff
+required_capability: to_date_nanos
+
+FROM date_nanos
+| EVAL n = MV_MAX(nanos)
+| EVAL diff_sec = DATE_DIFF("seconds", TO_DATE_NANOS("2023-10-23T12:15:03.360103847Z"), n)
```

Comment:
Should we add two extra cases for the combinations of millis and nanos? Maybe in this same test, as extra columns

Reply:
I've added one such.